File: URI_Online_Judge_1078.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {

		int N, Multiplication;

		Scanner input = new Scanner(System.in);

		N = input.nextInt();
		for (int i = 1; i <= 10; i++) {
			Multiplication = i * N;
			System.out.println(i + " x " + N + " = " + Multiplication);

		}
	}

}

File: URI_Online_Judge_1078.c
#include <stdio.h>
 
int main() {
 
    int N,i,Multiplication;
    scanf("%d",&N);
    
    for(i=1; i<=10; i++){
    	
    	Multiplication = i * N;
    	printf("%d x %d = %d\n",i,N,Multiplication);
    	
	}
     
    return 0;
}
